source/routing/next_stop_time.cpp

```diff
@@ -59,10 +59,10 @@ bool NextStopTimeData::Arrival::is_valid(const type::StopTime& st) const {
 template <typename Getter>
 void NextStopTimeData::TimesStopTimes<Getter>::init(const JourneyPattern& jp, const JourneyPatternPoint& jpp) {
     // collect the stop times at the given jpp
-    const size_t jpp_order = jpp.order;
+    const auto jpp_order = jpp.order;
```

Comment:
cheap trick for ya :smile: 

```suggestion
    const auto jpp_order = jpp.order.val;
```